pkg/syncer/util.go

```diff
@@ -361,6 +361,7 @@ func getConfig(ctx context.Context) (*cnsconfig.Config, error) {
 				"vSphere config secret and in immutable ConfigMap")
 		}
 		cfg.Global.ClusterID = clusterID
+		cnsconfig.GeneratedVanillaClusterID = clusterID
```

Comment:
we don't need to set cnsconfig.GeneratedVanillaClusterID  for syncer.
syncer is shared between vanilla and supervisor
and when you need to create VC session `cfg.Global.ClusterID` should be already set.

Reply:
fixed in latest commit